config/software/curl.rb

```diff
@@ -30,7 +30,7 @@
 
   if freebsd?
     # from freebsd ports - IPv6 Hostcheck patch
-    patch source: "curl-freebsd-hostcheck.patch", plevel: 1
+    patch source: "curl-freebsd-hostcheck.patch", plevel: 1, env: env
```

Comment:
this really smells like we should just bake this into the definition of `#patch`, and i believe we've talked before about `#command` and pretty much every other command should pick up env by default (if its defined).